src/components/styles/YouTube.scss

```
@@ -0,0 +1,94 @@
+@import 'variables';
+
+#intro-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: auto;
+}
+
+#intro-title {
+  font: $font-title;
+  margin-bottom: 0;
+  color: #fff;
+}
+
+#intro-subtitle {
+  font: $font-title;
+  font-size: 26px;
+  margin: 0;
+  color: #fff;
+}
+
+#youtube-computer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  background-image: url("../../assets/computer-youtube.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
+#youtube-wrapper {
+  margin-left: 10%;
+  margin-right: 10%;
+  width: 50%;
```

since we changed youtube computer, change youtube wrapper accordingly
```suggestion
  margin-left: 3vw;
  margin-right: 3vw;
  width: 100%;
```